utils/cliutils/commandsflags.go

```diff
@@ -734,6 +741,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  regexpFlag,
 		Usage: "[Default: false] Set to true to use a regular expression instead of wildcards expression to collect files to be added to the build info.` `",
 	},
+	badAnt: cli.BoolFlag{
```

Comment:
It looks like this flag hasn't been added to the ```BuildAddDependencies``` slice, which includes all the flags of the ```build-add-dependencies``` command.